profiles/common/modules/features/nexteuropa_editorial/includes/nexteuropa_editorial.tracking.inc

```diff
@@ -0,0 +1,429 @@
+<?php
+/**
+ * @file
+ * Code specific to customization for the change tracking management.
+ */
+
+/**
+ * Field widget form alter callback.
+ *
+ * It inject a specific validate callback in achrge of scanning field value.
```

Comment:
Maybe like this: (there is also typo `in charge of`)
Injects a custom validation callback which is searching for change tracking markup in the field value.